### PR TITLE
Update benchmark script

### PR DESCRIPTION
**Pull request by simonsso**
Broke out the benchmark changes from PR #753

### CODE REVIEW

File: pallets/reserve/src/lib.rs [hunk 1]
```diff
@@ -20,6 +20,7 @@
 
 //! A module that is called by the `collective` and is in charge of holding
 //! the company funds.
+#[cfg(feature = "runtime-benchmarks")]
```
Comment (ETeissonniere) [inline pallets/reserve/src/lib.rs:23]: I do not think this necessary as it is already at the top of `benchmarking.rs`?

File: pallets/reserve/src/benchmarking.rs [hunk 2]
```diff
@@ -37,15 +37,19 @@ const SEED: u32 = 0;
...
+		let starting_amount = smallest_usable_amount.saturating_add(smallest_usable_amount);
+		let tip_amount = smallest_usable_amount;
+		let _ = T::Currency::make_free_balance_be(&tipper,starting_amount);
+	}: _(RawOrigin::Signed(tipper), tip_amount)
 
 	spend {
 		let dest = account("dest", 0, SEED);
-		let value = 10u32.into();
+		let smallest_usable_amount = T::Currency::minimum_balance();
+		let starting_amount = smallest_usable_amount.saturating_add(smallest_usable_amount);
```
Comment (ETeissonniere) [inline pallets/reserve/src/benchmarking.rs:49]: It appear the original code wasn't created a balance superior to amount spent. As such, I do not think we need this line of code

Comment (simonsso) [reply pallets/reserve/src/benchmarking.rs:49]: Existing Deposits is enforced in tests now, I think the old code was broken.

I think it should spend some tokens here and update one storage item to get a heavier weight.

File: pallets/reserve/src/benchmarking.rs [hunk 3]
```diff
@@ -37,15 +37,19 @@ const SEED: u32 = 0;
...
+		let value = smallest_usable_amount;
```
[the remark this answers is not in the document]
Comment (simonsso) [reply pallets/reserve/src/benchmarking.rs:50]: Removed

File: pallets/reserve/src/lib.rs (hunk 1 above)
Comment (simonsso) [reply pallets/reserve/src/lib.rs]: I will double check

**Review by aliXsed — APPROVED** [review]

**Pull request merged**